misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingTreeModifier.java

```diff
@@ -281,7 +282,14 @@ public FunctionDefinitionNode transform(FunctionDefinitionNode functionDefinitio
         MetadataNode metadata = formatNode(functionDefinitionNode.metadata().orElse(null), 0, 1);
```

Comment:
No changes required to the `MethodDeclarationNode`?

Reply:
This [1] seems to be added in the `MethodDeclarationNode`.

[1] https://github.com/ballerina-platform/ballerina-lang/blob/f919b1d55a84b1d302f188e657cf5155755c0327/compiler/ballerina-parser/src/main/java/io/ballerina/compiler/syntax/tree/TreeModifier.java#L2257

Reply:
Fixed it here: 

https://github.com/ballerina-platform/ballerina-lang/blob/28c7bcab80ae1b8988019c7beee6053d04290524/misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingTreeModifier.java#L2744